package.json

```diff
@@ -51,16 +54,21 @@
     "rollup-plugin-polyfill-node": "^0.13.0",
     "sharedb": "^3.2.1",
     "sinon": "^17.0.1",
-    "typedoc": "^0.25.12",
-    "typedoc-plugin-mdn-links": "^3.1.18",
+    "tslib": "^2.8.1",
+    "typedoc": "^0.26.5",
+    "typedoc-plugin-extras": "^3.1.0",
+    "typedoc-plugin-mdn-links": "^3.2.6",
+    "typedoc-plugin-missing-exports": "^3.0.0",
+    "typedoc-plugin-rename-defaults": "^0.7.1",
     "typescript": "^5.3.3"
   },
   "scripts": {
     "test": "karma start test/karma.conf.js --single-run",
     "test:debug": "karma start test/karma.conf.js --single-run=false --browsers Chrome",
-    "build": "rollup -c",
-    "build:types": "rollup -c --environment target:types",
+    "build": "tsc",
+    "watch": "tsc -w",
     "docs": "typedoc",
-    "lint": "eslint --ext .js,.mjs src"
+    "lint": "eslint src",
+    "lint-fix": "eslint src --fix"
```

Comment:
We use colons rather than dashes for script names. I kind of want to try to share as many same names across all our package.json files as possible, ideally. We haven't had a specific one for `lint:fix` before - although not super-averse to it.